spec/rubocop/cop/rails/find_each_spec.rb

```diff
@@ -33,6 +33,7 @@
   it_behaves_like('register_offense', 'unscoped')
   it_behaves_like('register_offense', 'where(name: name)')
   it_behaves_like('register_offense', 'where.not(name: name)')
+  it_behaves_like('register_offense', 'where(name: name).or(User.where(age: age))')
```

Comment:
Can you move to between L30 (`left_outer_joins`) and L31 (`preload`)?

Reply:
I fixed it!.

Reply:
https://github.com/rubocop/rubocop-rails/pull/995/files#diff-8b9e38f28a7bdb108efab88f1f3495067c22717be7973a6c73e92d5b0f599287L35
If you want to maintain the order defined in `SCOPE_METHODS`, should I move this between `left_outer_joins` and `or`?

Reply:
Let's leave the existing one in favor of keeping `git blame`. Thank you.